packages/types/src/promotion/service.ts

```diff
@@ -64,4 +68,28 @@ export interface IPromotionModuleService extends IModuleService {
     rulesData: RemovePromotionRuleDTO[],
     sharedContext?: Context
   ): Promise<PromotionDTO>
+
+  createCampaigns(
+    data: CreateCampaignDTO[],
+    sharedContext?: Context
+  ): Promise<CampaignDTO[]>
```

Comment:
suggestion: Support single campaign creation 
```suggestion
  createCampaigns(
    data: CreateCampaignDTO[],
    sharedContext?: Context
  ): Promise<CampaignDTO[]>
  
    createCampaigns(
    data: CreateCampaignDTO,
    sharedContext?: Context
  ): Promise<CampaignDTO>
```